app/helpers/application_helper.rb

```diff
@@ -389,4 +389,12 @@ def hmis_enabled?
   def hmis_admin_visible?
     HmisEnforcement.hmis_admin_visible?(current_user)
   end
+
+  def omni_auth_providers
+    if ENV['ENABLE_HMIS_API']
+      [['Okta', '/users/auth/okta']]
+    else
```

Comment:
I'm probably missing something, but not _all_ HMIS installations will have an OKTA provider, only one at the moment, so that should probably be pulling from a database table or ENV.

Reply:
Thanks for spotting that, looks like I fat-fingered the wrong env variable there, my intention was `if ENV['OKTA_DOMAIN']`

This is is a replacement for  `resource_class.omniauth_providers` in the warehouse UI. If we need to move the provider config to the db we should check in on that since I've probably missed part of the spec

Reply:
Nope, `OKTA_DOMAIN` makes way more sense to me.